assignment.rb

```diff
@@ -0,0 +1,310 @@
+# The User class describes an account holder.
+class User
+  attr_accessor :name
+  attr_accessor :pin
+  attr_accessor :balance
+
+  def initialize
+    self.name = ""
+    self.pin = ""
+    self.balance = 0
+  end
+
+  # Verify that the user can withdraw the requested amount
+  def can_withdraw?(amount)
+    if self.balance >= amount
+      return true
+   else
+      return false
+   end
```

Comment:
as we spoke about yesterday, this could be reduced to 
```ruby
self.balance >= amount
```
because that already returns true or false!